Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "rust_hive"
+version = "0.1.0"
+authors = ["Patricio Tula <tula.patricio@gmail.com>", "Julian Mateu <julianmateu@gmail.com>"]
+edition = "2018"
```

Comment:
nit: we are in 2019

Reply:
According to [the docs](https://doc.rust-lang.org/cargo/reference/manifest.html#the-edition-field-optional), this refers to the Rust edition, and since it was created by default I believe it is OK using it (I'm asuming it is the latest version as explained there).

Reply:
ignore the comment then.